recipes/bpipe/meta.yaml

```diff
@@ -10,12 +10,12 @@ source:
   sha256: 'f640ab3c32ba8393b47a0019f5a9d3d168da1d741f5f2690797080037a6ce374'
 
 build:
-  number: 0
+  number: 1
   noarch: generic
 
 requirements:
   run:
-    - openjdk
+    - openjdk =11
```

Comment:
do later version work?

Reply:
Thanks for reviewing @bgruening. According to the author, the latest version of bpipe (0.9.11) supports JDK 11 but not beyond (see https://github.com/ssadedin/bpipe/issues/274 and https://github.com/Oshlack/MINTIE/issues/26). Official support for newer JDKs is in the works but currently not supported.